tcks/microprofile-fault-tolerance/src/test/java/io/quarkus/tck/faulttolerance/TimeoutUninterruptableTest.java

```diff
@@ -0,0 +1,267 @@
+package io.quarkus.tck.faulttolerance;
+
+import static org.eclipse.microprofile.fault.tolerance.tck.util.Exceptions.expect;
+import static org.eclipse.microprofile.fault.tolerance.tck.util.Exceptions.expectTimeout;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.lessThan;
+import static org.testng.Assert.*;
+import static org.testng.Assert.fail;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicLong;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.fault.tolerance.tck.timeout.clientserver.UninterruptableTimeoutClient;
+import org.eclipse.microprofile.fault.tolerance.tck.util.Packages;
+import org.eclipse.microprofile.faulttolerance.exceptions.BulkheadException;
+import org.eclipse.microprofile.faulttolerance.exceptions.TimeoutException;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+/**
+ * Customized test from MP FT TCK because `testTimeout` fails because Hystrix is unable to interrupt the calling thread, as it
+ * didn't create it.
+ * https://github.com/eclipse/microprofile-fault-tolerance/issues/408
+ */
+public class TimeoutUninterruptableTest extends Arquillian {
```

Comment:
Why don't we instead keep the exclude list with the precise method?
Because what if there is a change/addition on MP side? Then this class will be outdated and pretty much worthless....